MODULE_NOT_FOUND superagent(-proxy) are not defined as dependencies

Version of 3.1.0 of the SDK is not usable at this point, because for some reason superagent and superagent-proxy are no longer listed as dependencies, but only as devDependencies.

Immediately upon `require('code-push')` an error is thrown.

```
Error: Cannot find module 'superagent-proxy'
Require stack:
- .../node_modules/code-push/script/management-sdk.js
- .../node_modules/code-push/script/index.js
- .../scripts/get-latest-codepush-release.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (.../node_modules/code-push/script/management-sdk.js:22:18)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '.../node_modules/code-push/script/management-sdk.js',
    '.../node_modules/code-push/script/index.js',
    '.../scripts/get-latest-codepush-release.js'
  ]
}
```

3.0.1 works fine.

--- a/package.json
+++ b/package.json
@@ -26,6 +26,8 @@
     "q": "^1.4.1",
     "recursive-fs": "^1.1.2",
     "slash": "^3.0.0",
+    "superagent": "^5.1.2",
+    "superagent-proxy": "^2.0.0",
     "yazl": "^2.4.1"
   },
   "devDependencies": {
@@ -37,9 +39,7 @@
     "@types/yazl": "^2.4.2",
     "mocha": "^6.2.2",
     "shx": "^0.3.2",
-    "superagent": "^5.1.2",
     "superagent-mock": "^3.7.0",
-    "superagent-proxy": "^2.0.0",
     "typescript": "^3.7.3"
   }
 }

